src/link/MachO/Atom.zig

```diff
@@ -482,8 +482,7 @@ pub fn scanRelocs(self: Atom, macho_file: *MachO) !void {
                 const symbol = rel.getTargetSymbol(self, macho_file);
                 if (symbol.flags.import or
                     (symbol.flags.@"export" and symbol.flags.weak) or
-                    symbol.flags.interposable or
-                    macho_file.getTarget().cpu.arch == .aarch64) // TODO relax on arm64
```

Comment:
You will need to implement GOT relaxation for this to work, which I haven't done for aarch64 (only for x86_64).

Reply:
Yeah I wasn't sure if that had actually been implemented. Not sure if I'll take a stab it, but just out of curiosity, can you point me to the relevant code?

Reply:
Sure thing. There are two steps to it:
1. applying this PR
2. working out instruction rewrites in https://github.com/ziglang/zig/blob/master/src/link/MachO/Atom.zig#L762-L785 and https://github.com/ziglang/zig/blob/master/src/link/MachO/Atom.zig#L815-L821 where the target will be `S + A -P` instead of `G + A - P` (kinda like it's done for x86_64 in https://github.com/ziglang/zig/blob/d8d2aa9af438bffa643aabb180011f275afd87d3/src/link/MachO/Atom.zig#L730-L740)

Reply:
Feel free to ping me for a review or whatnot should you need it!